react/src/components/SessionListColums/SessionInfoCell.tsx

```diff
@@ -90,62 +88,59 @@ const SessionInfoCell: React.FC<{
   // sessions[objectKey].icon = this._getKernelIcon(session.image);
   //         sessions[objectKey].sessionTags = this._getKernelInfo(session.image);
   return (
-    <Flex direction="column">
-      <Flex direction="row">
-        <Form form={form}>
-          {editing ? (
-            <Form.Item
-              style={{ margin: 0 }}
-              name={'name'}
-              rules={[
-                {
-                  required: true,
-                },
-                {
-                  max: 64,
-                },
-                {
-                  pattern: /^(?:[a-zA-Z0-9][-a-zA-Z0-9._]{2,}[a-zA-Z0-9])?$/,
-                  message: t(
-                    'session.Validation.EnterValidSessionName',
-                  ).toString(),
-                },
-              ]}
-            >
-              <Input
-                autoFocus
-                onPressEnter={() => save()}
-                onKeyUp={(e) => {
-                  if (e.key === 'Escape') setEditing(false);
-                }}
-              />
-            </Form.Item>
-          ) : (
-            <>
-              <Typography.Text style={{ opacity: isPendingRename ? 0.5 : 1 }}>
-                {optimisticName}
-              </Typography.Text>
-              {editable && (
-                <Button
-                  loading={isPendingRename}
-                  type="text"
-                  icon={<EditOutlined />}
-                  style={{ color: token.colorLink }}
-                  onClick={() => {
-                    form.setFieldsValue({
-                      name: session.name,
-                    });
-                    setEditing(true);
-                  }}
-                ></Button>
-              )}
-            </>
+    <Form form={form}>
+      {editing ? (
+        <Form.Item
+          style={{ margin: 0 }}
+          name={'name'}
+          rules={[
+            {
+              required: true,
+            },
+            {
+              max: 64,
+            },
+            {
+              pattern: /^(?:[a-zA-Z0-9][-a-zA-Z0-9._]{2,}[a-zA-Z0-9])?$/,
+              message: t('session.Validation.EnterValidSessionName').toString(),
```

Comment:
```suggestion
              message: t('session.Validation.EnterValidSessionName'),
```
It's already string.